spec/tapioca/dsl/compilers/active_record_associations_spec.rb

```diff
@@ -936,7 +936,7 @@ def pictures=(value); end
                   RUBY
 
                   add_ruby_file("user.rb", <<~RUBY)
-                    class User
+                    class User < ActiveRecord::Base
```

Comment:
How did the parent appear?

Reply:
It was supposed to always be there, actually, since `User` is supposed to be an AR model. On Rails 7, there seems to be a new check for associations that makes sure that the target of an association is a subclass of `ActiveRecord::Base`. If it is not you get this error:
```
Minitest::UnexpectedError:         ArgumentError: Rails couldn't find a valid model for User association. Please provide the :class_name option on the association declaration. If :class_name is already provided, make sure it's an ActiveRecord::Base subclass.
```
Ref: [this failure](https://github.com/Shopify/tapioca/runs/6214716337?check_suite_focus=true#step:8:171)